src/components/JobsView/JobsView.js

```diff
@@ -0,0 +1,175 @@
+import PropTypes from 'prop-types';
+import {
+  Button,
+  MultiColumnList,
+  Pane,
+  PaneMenu,
+  Paneset,
+} from '@folio/stripes-components';
```

Comment:
Hi, could you change imports of `stripes-core`, `stripes-components` and `stripes-smart-components` to be
```
from '@folio/stripes-components`
``` 
and
```
from '@folio/stripes/smart-components`
```

These modules don't come as peerDependencies, so they should be imported from `stripes` module directly
Otherwise it results in errors during build time
![image](https://user-images.githubusercontent.com/19309423/196409311-32ecc5bb-5b43-4098-bc7c-208ca7dcf7b9.png)